tests/baselines/reference/tsserver/partialSemanticServer/syntactic-diagnostics-are-returned-with-no-error.js

```diff
@@ -0,0 +1,31 @@
+Provided types map file "/a/lib/typesMap.json" doesn't exist
+request:{"seq":0,"type":"request","command":"open","arguments":{"file":"/user/username/projects/myproject/a.ts"}}
+Plugins were requested but not running in environment that supports 'require'. Nothing will be loaded
+Starting updateGraphWorker: Project: /dev/null/inferredProject1*
+Finishing updateGraphWorker: Project: /dev/null/inferredProject1* Version: 1 structureChanged: true structureIsReused:: Not Elapsed:: *ms
+Project '/dev/null/inferredProject1*' (Inferred)
+	Files (2)
+	/a/lib/lib.d.ts
+	/user/username/projects/myproject/a.ts
+
+
+	a/lib/lib.d.ts
+	  Default library for target 'es5'
+	user/username/projects/myproject/a.ts
+	  Root file specified for compilation
+
+-----------------------------------------------
+Project '/dev/null/inferredProject1*' (Inferred)
+	Files (2)
+
+-----------------------------------------------
+Open files: 
+	FileName: /user/username/projects/myproject/a.ts ProjectRootPath: undefined
+		Projects: /dev/null/inferredProject1*
+response:{"responseRequired":false}
+request:{"type":"request","seq":1,"command":"syntacticDiagnosticsSync","arguments":{"file":"/user/username/projects/myproject/a.ts"}}
+response:{"response":[{"start":{"line":1,"offset":17},"end":{"line":1,"offset":18},"text":"')' expected.","code":1005,"category":"error"}],"responseRequired":true}
+request:{"command":"geterr","arguments":{"delay":0,"files":["/user/username/projects/myproject/a.ts"]},"seq":2,"type":"request"}
+response:{"responseRequired":false}
+Session does not support events: ignored event: {"seq":0,"type":"event","event":"syntaxDiag","body":{"file":"/user/username/projects/myproject/a.ts","diagnostics":[{"start":{"line":1,"offset":17},"end":{"line":1,"offset":18},"text":"')' expected.","code":1005,"category":"error"}]}}

```

Comment:
How hard would it be to support events?   Technically, this is sufficient for baselining purposes, but it's a bit ugly.

Reply:
background question: what is an event, and does the command syntacticDiagnosticsSync trigger the syntaxDiag event?

Reply:
An event is a message from the server that is not a direct response to a request.  Common examples are state changes (type acquisition completed, project loading started, etc) and diagnostics.  Editors like VS Code that send `getErr` requests expect to subsequently receive `syntaxDiag` (etc) events asynchronously providing the answer to their question.  Editors like VS that send `syntacticDiagnosticsSync` expect to receive the the answer in the corresponding response message.

Reply:
FWIW, if you do go in and get the `TestSession` to support events, please leave the ability for a `Session` to not necessarily support events - there's no analogous concept in LSP, so it's convenient to leave that factored out

Reply:
I'm not planning to start supporting events in this PR - the whole point is just to baseline the output of the event.